plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1215,10 +1215,18 @@ blockly.block.set_event_result.line2=Works only with global triggers with a resu
 blockly.block.call_procedure=Call procedure
 blockly.block.procedure_retval=Call procedure and get return value
 blockly.block.call_procedure.at=\ at
+blockly.block.block_predicate_all_of=All conditions are met
+blockly.block.block_predicate_all_of.empty=(Always)
```

Comment:
Do those actually work this way, so if list of all_of predicate it empty, it will default to true?

Reply:
Yes, you can check `AllOfPredicate` and `AnyOfPredicate`

Reply:
Did not have time to do it yesterday and asked, thanks for the clarification